Packaging.props

```diff
@@ -4,11 +4,6 @@
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
     <PackageLicenseFile>$(ProjectDir)pkg/dotnet_library_license.txt</PackageLicenseFile>
     <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
-    
-    <!-- *** start WCF Content *** -->
-    <!-- *** this file is located differently in corefx *** -->
-    <RuntimeIdGraphDefinitionFile>$(PackagesDir)Microsoft.NETCore.Platforms/1.0.1-rc3-24206-00/runtime.json</RuntimeIdGraphDefinitionFile>
-    <!-- *** end WCF Content *** -->
```

Comment:
Per Eric's commit and @StephenBonikowsky - this is now in buildtools, so this no longer needs to appear in our Packaging.props